src/ty/path.rs

```diff
@@ -186,19 +201,19 @@ mod tests {
     #[test]
     fn path_ok() {
         assert_eq!(
-            Path::from_segments(vec!["hello"]),
+            Path::<MetaForm>::from_segments(vec!["hello"]),
```

Comment:
I guess this highlights that making the functions no longer be only on `MetaForm`, code could break since we need the extra type information sometimes.

Reply:
I've moved these back to MetaForm now so should be non-breaking